README.md

```diff
@@ -2,6 +2,39 @@
 
 GTK4 Revolt client
 
+## Installation
+
+First, make sure the following components are installed:
+
+* (The meson build system)[https://mesonbuild.com/Quick-guide.html]
+* (The Rust toolchain)[https://www.rust-lang.org/tools/install]
+* (The GTK4 library)[https://github.com/ToshioCP/Gtk4-tutorial/blob/main/gfm/sec2.md]
+* (The Adwaita library)[https://github.com/GNOME/libadwaita]
+
+After installing the necessary components, set up `builddir`:
+```sh
+meson setup builddir
+```
+
+Once this is done, compile the app:
+```sh
+cd builddir && meson compile
+```
+
+Unfortunely, in some cases it is difficult to utilize `meson install` properly. In this case the user can just specify the prefix to the current build directory:
+```sh
+# inside builddir
+meson configure -D `pwd`
```

Comment:
I'm not sure what this is supposed to do. The `-D` command-line flag. expects to receive an argument such as `option_name=value`, and you're giving it the current directory which is a value without an option name.

Guessing based on the context of "the `share` folder on the build directory" that this is supposed to be `-D prefix=$(pwd)`?

Reply:
It seems to me like you probably don't want to do any of this. Instead I would investigate https://mesonbuild.com/Commands.html#devenv which you can also use in combination with `meson.add_devenv()` to enhance the devenv shell with e.g. XDG_DATA_DIRS, use gnome.compile_schemas(), and so on.